lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -237,26 +237,45 @@ class ConvertAtenViewOp : public OpConversionPattern<AtenViewOp> {
                              int64_t startExpandDim, int64_t maxExpandDim,
                              const SmallVector<int64_t> &collapseShape,
                              const SmallVector<int64_t> &expandShape,
-                             ReassociationIndices &expandIndices) {
+                             ReassociationIndices &collapseIndices,
+                             ReassociationIndices &expandIndices
+                             ) {
+    int64_t expandDimSize = expandShape[startExpandDim];
```

Comment:
nit: this variable is unnecessary since you start with expandedSize = 1;
you can just put the declaration down below where you use it.